packages/rum-core/test/performance-monitoring/performance-monitoring.spec.js

```diff
@@ -241,35 +240,30 @@ describe('PerformanceMonitoring', function() {
 
   it('should filter transactions based on browser responsiveness', function() {
     configService.setConfig({
-      browserResponsivenessInterval: 500,
-      checkBrowserResponsiveness: true,
-      browserResponsivenessBuffer: 2
+      checkBrowserResponsiveness: true
     })
     spyOn(logger, 'debug').and.callThrough()
     expect(logger.debug).not.toHaveBeenCalled()
     var tr = new Transaction('transaction', 'transaction', {
       id: 212,
       transactionSampleRate: 1,
       managed: true,
-      checkBrowserResponsiveness: true
+      checkBrowserResponsiveness: true,
+      startTime: 1
     })
     var span = tr.startSpan('test span', 'test span type')
     span.end()
-    tr.end()
-    tr._start = 1
+    tr.end(3501)
 
-    tr._end = 3001
     tr.browserResponsivenessCounter = 3
     var wasBrowserResponsive = performanceMonitoring.filterTransaction(tr)
     expect(wasBrowserResponsive).toBe(false)
     expect(logger.debug).toHaveBeenCalledWith(
       'transaction(212, transaction) was discarded! Browser was not responsive enough during the transaction.',
       ' duration:',
-      3000,
+      3500,
       ' browserResponsivenessCounter:',
-      3,
-      'interval:',
-      500
+      3
     )
   })
```

Comment:
Same as above for this test

Reply:
This test depends on configs which are removed, so it does not apply here.